#! /usr/bin/env python
PACKAGE='opencv_example'
from dynamic_reconfigure.msg import SensorLevels
from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()

gen.add("blue_thres",  int_t, 0, "Threshold",   200, 0, 255)
gen.add("erode_size",  int_t, 0, "Erode Size",  3, 1, 21)
gen.add("dilate_size",  int_t, 0, "Dilate Size", 3, 1, 21)
gen.add("hough_rho_res",  int_t, 0, "Rho resolution (pix)", 10, 1, 500)
gen.add("hough_theta_res",  double_t, 0, "Theta resolution (rad)", 0.05, 0.01, 1.0)
gen.add("hough_threshold",  int_t, 0, "Accumulator threshold", 25, 10, 200)
gen.add("hough_min_length",  int_t, 0, "Minimum line length (pix)", 250, 5, 500)
gen.add("hough_max_gap",  int_t, 0, "Max gap betweeen line segments (pix)", 50, 5, 500)

exit(gen.generate(PACKAGE, PACKAGE, "RgbHough"))
